GameData/RP-0/changelog.cfg

```
@@ -24,13 +24,14 @@ KERBALCHANGELOG
 			subchange = should move tag
 		}
 	}
+	
 	VERSION
 	{
 		version = 0.0.0.5
 		versionKSP = 1.12.3
 		CHANGE
 		{
-			change = 
+			change = test
 		}
 	}
```

Auto-generated potential formatting fix:
```suggestion
		versionKSP = 1.12.3
		CHANGE { change = test }
	}
```